recipes/sphinx-jsonschema/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sphinx-jsonschema" %}
+{% set version = "1.16.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinx-jsonschema-{{ version }}.tar.gz
+  sha256: 5728b3ece1fe475c285d5ae16953242c51b4e6852f48f0bb734cd8408f3ac8b3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - docutils
+    - jsonpointer
+    - pyyaml
+    - requests
+
+test:
+  # There are no import tests because this package provides no imports, just a sphinx plugin.
+  # There are currently no functional unit tests ( https://github.com/lnoor/sphinx-jsonschema/issues/56 ).
```

Comment:
You should still be able to import `sphinx-jsonschema`. That's what `sphinx` will do under the hood, and it will validate that the package isn't broken.

Reply:
It's a bit tricky with the dash in the name, but I can do it with `python -c` and `importlib`.